astropy/coordinates/tests/test_solar_system.py

```diff
@@ -125,16 +120,17 @@ class TestPositionsGeocentric:
     def setup(self):
         self.t = Time('1980-03-25 00:00')
         self.frame = GCRS(obstime=self.t)
```

Comment:
Don't think `self.frame` is used any more (but may have missed it...)

Reply:
good catch!